src/otx/core/data/module.py

```diff
@@ -37,18 +38,26 @@ def __init__(self, task: OTXTaskType, config: DataModuleConfig) -> None:
         self.subsets: dict[str, OTXDataset] = {}
         self.save_hyperparameters()
 
-        dataset = DmDataset.import_from(
-            self.config.data_root,
-            format=self.config.data_format,
-        )
+        if self.config.data_format == "common_semantic_segmentation":
+            # temporary hack, should be fixed later
+            dataset = {}
+            train_data_roots = Path(self.config.data_root) / "train"
+            val_data_roots = Path(self.config.data_root) / "val"
+
+            for subset_name, subset_path in zip(("train", "val", "test"),
+                                           (train_data_roots, val_data_roots, val_data_roots)):
+                dataset[subset_name] = DmDataset.import_from(subset_path,
+                                                     format=self.config.data_format).subsets()["default"]
+        else:
+            dataset = DmDataset.import_from(self.config.data_root, format=self.config.data_format).subsets()
 
         config_mapping = {
             self.config.train_subset.subset_name: self.config.train_subset,
             self.config.val_subset.subset_name: self.config.val_subset,
             self.config.test_subset.subset_name: self.config.test_subset,
         }
 
-        for name, dm_subset in dataset.subsets().items():
+        for name, dm_subset in dataset.items():
```

Comment:
I'm not sure this change is valid for other tasks. Could you rebase so we can run integration tests for this PR?